src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/config/BvlsRequestContextConfiguration.kt

```diff
@@ -32,36 +35,27 @@ class BvlsRequestContextConfiguration(private val bvlsRequestContextInterceptor:
 }
 
 @Configuration
-class BvlsRequestContextInterceptor : HandlerInterceptor {
-
-  companion object {
-    private val log = LoggerFactory.getLogger(BvlsRequestContextInterceptor::class.java)
-  }
+class BvlsRequestContextInterceptor(private val userService: UserService) : HandlerInterceptor {
 
   override fun preHandle(request: HttpServletRequest, response: HttpServletResponse, handler: Any): Boolean {
-    val username = getUsernameFromClaim()
+    val username = authentication().userName
```

Comment:
I like this.  I didn't realise there were specific methods for user name and client id.